django/initial_setup.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+echo "Installing Python dependencies..."
+pip install -r requirements.txt || { echo "Error: Pip install failed"; exit 1; }
```

Comment:
This is taken care of in the docker file already no?

Reply:
Oops, meant to remove it earlier.